webapp/src/hooks/index.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import { isDesktopApp } from 'src/utils/user_agent';
+import { connectUsingBrowserMessage } from 'src/constants';
+import {sendEphemeralPost} from '../actions';
+
+type ContextArgs = {channel_id: string};
+
+const connectCommand = '/gitlab connect';
+
+export default class Hooks {
+    private store: any;
+
+    constructor(store: any) {
+        this.store = store;
+    }
+
+    slashCommandWillBePostedHook = (rawMessage: string, contextArgs: ContextArgs) => {        
```

Comment:
have we tested for all other commands?

Reply:
Yes, the other commands are not affected